src/vector.rs

```diff
@@ -151,6 +150,58 @@ impl<T, U> Vector2D<T, U> {
     pub fn from_untyped(p: Vector2D<T, UnknownUnit>) -> Self {
         vec2(p.x, p.y)
     }
+
+    /// Computes the vector with absolute values of each component.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// # use std::{i32, f32};
+    /// # use euclid::vec2;
+    /// enum U {}
+    ///
+    /// assert_eq!(vec2::<_, U>(-1, 2).abs(), vec2(1, 2));
+    ///
+    /// let vec = vec2::<_, U>(f32::NAN, -f32::MAX).abs();
+    /// assert!(vec.x.is_nan());
+    /// assert_eq!(vec.y, f32::MAX);
+    /// ```
+    ///
+    /// # Panics
+    ///
+    /// Method panics if `abs()` of underlying type panics:
+    ///
+    /// ```rust,should_panic
```

Comment:
This only panics debug builds, you are adding a test that implies that abs always panics with . Please just remove this section, it is too specific and describes something that is outside of euclid. If you really want to be explicit about the behavior of abs, you can write "The behavior for each component follows the scalar type's implementation of `num_traits::Signed::abs`."

Reply:
Ok, done.